genealogy/gramps5/Portfile

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+name                    gramps5
+github.setup            gramps-project gramps 5.0.1 v
+revision                0
+
+PortGroup               python 1.0
+
+license                 GPL-2
+categories              genealogy python
+platforms               darwin
+supported_archs         noarch
+maintainers             {devans @dbevans} openmaintainer
+           
+description             Gramps is a genealogy program
+
+long_description        Gramps is a genealogy program to store, edit, \
+                        and research genealogical data.  It provides \
+                        advanced capabilities for research, analysis, and \
+                        correlation to potentially fill relationship gaps. This \
+                        version is based on gtk3 and supports both python 2 and 3. \
+                        To use gramps 4.x install port gramps. To use the legacy \
+                        gtk2 version, install port gramps3.
+
+homepage                https://www.gramps-project.org/
+
+checksums               rmd160  17440bcaafaf34f573777b65cb388ddc2fc07391 \
+                        sha256  c4ff143dc3132b0227328edb94458b1b70b87d7a7fb1452bafefbe174d220328 \
+                        size    16718002
+
+depends_build-append    port:intltool
+
+depends_lib-append      path:bin/dot:graphviz \
+                        port:desktop-file-utils \
+                        port:gexiv2 \
+                        port:ghostscript \
+                        port:goocanvas2 \
+                        port:gtk3 \
+                        port:gtkspell3 \
+                        port:osm-gps-map \
+                        port:shared-mime-info \
+                        port:xdg-utils
+
+depends_run             port:adwaita-icon-theme
```

Comment:
Change `depends_run` to `depends_run-append` just in case any runtime decencies come from the portgroups being used.